### ISSUE TITLE
Map shows black screen on Android 12 and 13 (API 31-33)

### ISSUE THREAD

**Issue by maxammann** [m0nac0/flutter-mapbox-gl#193]
I'm experiencing black screens when launching our app on Android 12 (Simulator) and Android 13 (Pixel 7) in release and debug mode.

Here is a video which shows that the screen goes black then rotating the device. Rotating seems to mimic an app resetart for maplibre.

https://user-images.githubusercontent.com/905221/216569729-c2acb2ea-e649-49c9-83bb-60aad7be24f4.mp4

I'm not sure if this is a maplibre-gl issue or flutter related.

Downstream issue: https://github.com/digitalfabrik/entitlementcard/issues/727

**Comment by maxammann** [comment]
Probably related to this issue: https://github.com/flutter-mapbox-gl/maps/issues/1180

**Comment by maxammann** [comment]
Did a git bisect to find this:

```
0.15.0 - good
f781c67ecfa3cec7d7e13585d5e39a2322796574 good
6cae6aeebf259cf742cbc8e178a037af6681eeca - good 
a26a4e03e40105e080d435005636ef5d0c651d20 - good
7a7e58ba43fc6caa188b0485bef8bcc22f136270 - good
6408bbbaa53a51925b9edecd534a6e5e002fa13f - good
79e929a4f47a31f0afa4b3ceb108c6e0ff264c7d - good
1275bb92f896035cd947eea0354957824715a9bf - good

f8b2d1c584c16a22d46d4c5200344d5159252895 - candidate

a4ad66ee7396cb423b79e025796313c514717b31 - bad
b7e07fde097ca11baadde17d5d9f8f9381d0238c - bad
379ed36eff823a95b2b3dff473f94bf661411774 - bad
0.15.1 - bad
0.16.0 - bad
```

**Comment by maxammann** [comment]
As it turns out this was already fixed here in the mapbox upstream: https://github.com/flutter-mapbox-gl/maps/pull/1216

**Comment by mariusvn** [comment]
looks like you're right, i think it also fixes the abandonned BufferQueue log, needs to be verified tho

**Comment by mariusvn** [comment]
is that fixed with the last edits @maxammann ? we re-enabled the texturemode

**Comment by maxammann** [comment]
yes, fixed by #194
